ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var subtitleNumberOfLines: Int = Constants.subtitleDefaultNumberOfLines {
+        didSet {
+            if subtitleNumberOfLines != oldValue {
+                secondaryLabel.numberOfLines = subtitleNumberOfLines
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var cardBackgroundColor: UIColor = Constants.defualtBackgroundColor {
+        didSet {
+            if cardBackgroundColor != oldValue {
+                setupBackgroundColor()
+            }
+        }
+    }
+
+    /**
+     Initializes `CardView`
+     
+     - Parameter style: The style of the card
+     - Parameter title: The title of the card
+     - Parameter subtitle: The subtitle of the card - optional
+     - Parameter icon: The icon of the card - optional
+     - Parameter image: The featured image of the card (small, medium, or announcement) - optional
+     - Parameter backgroundStyle: The Card's background style; appColor, neutral, or custom.
+     - Parameter buttonTitle: The button's title (announcement type only) - optional
+     **/
+    @objc public init(style: CardStyle, title: String, subtitle: String? = nil, icon: UIImage? = nil, image: UIImage? = nil, backgroundStyle: CardBackgroundStyle, buttonTitle: String? = nil) {
+        self.primaryText = title
+        self.style = style
+        self.secondaryText = subtitle
+        self.icon = icon
+        self.image = image
+        self.buttonTitle = buttonTitle
+        self.backgroundStyle = backgroundStyle
+
+        super.init(frame: .zero)
+
+        configureSubviews()
+    }
+
+    @available(*, unavailable)
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func configureSubviews() {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        // view border and background
+        layer.borderColor = Constants.borderColor.cgColor
+        layer.borderWidth = Constants.borderWidth
+        layer.cornerRadius = Constants.borderRadius
+        setupBackgroundColor()
+
+        if style != .announcement {
+            let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleCardTapped(_:)))
+            addGestureRecognizer(tapGesture)
+        }
+
+        // Title
+        primaryLabel.text = self.primaryText
+        primaryLabel.numberOfLines = titleNumberOfLines
+        primaryLabel.textAlignment = style.textAlignment
+        addSubview(primaryLabel)
+
+        // Subtitle
+        if let secondaryText = secondaryText {
+            secondaryLabel.text = secondaryText
+            secondaryLabel.numberOfLines = subtitleNumberOfLines
+            secondaryLabel.textAlignment = style.textAlignment
+            addSubview(secondaryLabel)
+        }
+
+        // Icon
+        if let icon = icon {
+            let iconView = UIImageView(image: icon)
```

Comment:
nit: this could be simplified to 

iconView = UIImageView(image: icon)

That way you can remove self.iconView = iconView

Reply:
Same for imageView below

Reply:
and button

Reply:
If I do that I can an error that the value of the optional must be unwrapped..